src/transformState.js

```diff
@@ -61,8 +61,33 @@
  * @param {Object} state
  * @param {Object[]} transforms
  */
+
 function transformState(state, transforms) {
-  // write code here
+  const linkToStateObj = state;
+
+  transforms.forEach((operationObj) => {
```

Comment:
```suggestion
  transforms.forEach(({ properties, operation }) => {
```
Можно использовать properties operation вместо operationObj.properties operationObj.operation.